org.gnome.pushupmaster.json

```diff
@@ -0,0 +1,52 @@
+{
+    "app-id" : "org.gnome.pushupmaster",
```

Comment:
It this an official gnome app?

Reply:
actually no, but I put it is on the git.gnome repo.
If I should change sth there, please tell me. It's my first app :-)

Reply:
Yup, this isn't GNOME software as released by the GNOME release team. Check out https://circle.gnome.org/ for ways to get your app a 'seal of approval'.

Reply:
Thanks for the hint. I wanted to do this anyway. Let me work on the other topics and some additional improvements on the SW and I will go for it.